tests/tests.py

```diff
@@ -209,3 +212,36 @@ def test_token_expiry_is_not_extended_within_MIN_REFRESH_INTERVAL(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(original_expiry, AuthToken.objects.get().expires)
+
+    def test_exceed_token_amount_per_user(self):
+
+        with override_settings(REST_KNOX=token_user_limit_knox):
+            reload_module(views)
+            for _ in range(10):
+                token = AuthToken.objects.create(user=self.user)
+            url = reverse('knox_login')
+            self.client.credentials(
+                HTTP_AUTHORIZATION=get_basic_auth_header(self.username, self.password)
+            )
+            response = self.client.post(url, {}, format='json')
+            reload_module(views)
+            self.assertEqual(response.status_code, 401)
+            self.assertEqual(response.data, {"error": "Maximum amount of tokens allowed per user exceeded."})
+
+    def test_does_not_exceed_on_expired_keys(self):
+
+        with override_settings(REST_KNOX=token_user_limit_knox):
+            #reload_module(auth)
```

Comment:
Should have `reload_module(views)`  otherwise you are not testing your feature.
Would add a sign up request after the successful one to test that it indeed fail after 10 valid tokens.

Reply:
@rphlo my bad, I went to fast. What release do you want me to add this feature under in `changes.md`?